src/main/java/com/dev/cinema/model/dto/MovieSessionRequestDto.java

```diff
@@ -1,6 +1,9 @@
 package com.dev.cinema.model.dto;
 
+import javax.validation.constraints.NotNull;
+
 public class MovieSessionRequestDto {
+    @NotNull
     private String showTime;
     private Long cinemaHallId;
     private Long movieId;
```

Comment:
show Time is not null but is it even possible to have a movie session with null cinema hall and movie?